tests/telemetry/test_host.py

```diff
@@ -0,0 +1,48 @@
+import platform
+
+import mock
+
+from ddtrace.internal.telemetry.data.host import HOST
+from ddtrace.internal.telemetry.data.host import Host
+from ddtrace.internal.telemetry.data.host import get_containter_id
+from ddtrace.internal.telemetry.data.host import get_hostname
+from ddtrace.internal.telemetry.data.host import get_os_version
+
+
+def test_host_fields():
+    expected_host = {
+        "os": platform.platform(aliased=1, terse=1),
+        "hostname": get_hostname(),
+        "os_version": get_os_version(),
+        "kernel_name": platform.system(),
+        "kernel_release": platform.release(),
+        "kernel_version": platform.version(),
+        "container_id": get_containter_id(),
+    }  # type: Host
+
+    assert HOST == expected_host
+
+
+def test_get_os_version_macos():
+    with mock.patch("platform.mac_ver") as macos:
+        macos.return_value = ("3.5.6", ("", "", ""), "")
+        assert get_os_version() == "3.5.6"
+
+
+def test_get_os_version_win32():
+    with mock.patch("platform.win32_ver") as win32:
+        win32.return_value = ("", "4.1.6", "", "")
+        assert get_os_version() == "4.1.6"
+
+
+def test_get_container_id():
+    # not sure how to mock get_container_info()
```

Comment:
take a look at the test cases for `get_container_info`.

you will need test cases for when `get_container_info` has results and does not have results.

Reply:
Yup the issue was I was trying to patch `ddtrace.internal.runtime.container.get_container_info` instead of `ddtrace.internal.telemetry.data.host.get_container_info`. I made that change and included the additional test